wordpress/wp-content/plugins/cds-base/classes/Modules/Contact/ContactForm.php

```diff
@@ -77,6 +77,15 @@ public function render($atts, $content = null): string
                         $heading = __('Tell us more', 'cds-snc');
                         $label = __('Your message', 'cds-snc');
                         break;
+                    default:
+                        $contactType = "";
+                }
+                ?>
+
+                <?php
+
+                if (!$contactType) {
+                    return __('Something Went Wrong Try Again.', 'cds-snc');
                 }
```

Comment:
Could we make this more sentence-y? Like "Something went wrong. Please try again."?